src/pages/library/dataBrowser-utils.js

```diff
@@ -112,13 +100,13 @@ const extractTags = dataset => {
     itemsType: 'AttributeValue',
     items: _.flow(_.flatten, _.toLower)([
       dataset.access,
-      _.map('dct:title', dataset['TerraDCAT_ap:hasDataCollection']),
+      getConsortiumsFromDataset(dataset),
```

Comment:
This is a small fix I figured I'd just roll in with this one.

Reply:
This makes me wonder, once this is further along, is the goal something like `getXFromDataset(dataset), getYFromDataset(dataset)` (or `dataset.getX()`, `dataset.getY()` which seems equivalent to me), OR something like `(X, Y).map(attr -> dataset.getAttribute(attr))`.

The generic `getAttribute()` approach would work if all attributes can support a common API. It has the potential benefit of making our UI even more data driven, where the key name for an attribute can be used when creating the details view, column headers and filter pills.

Reply:
This one I am less sure on the approach for. Maybe we can sync up in voice or screenshare post retreat on Wednesday?